ci/unit_tests/functions_delete_package/main_test.py

```diff
@@ -257,3 +292,34 @@ def test_wrongNamespace(self, useApikey):
         # Fail
         self.t_exitCodeAndLogMessage(1,
         "The resource could not be found. Check your cloudfunctions url and namespace.", [paramsDelete])
+
+    @pytest.mark.parametrize('useApikey', [True])
+    def test_deletePackageByRegex(self, useApikey):
```

Comment:
I would create one more package and test if it wasn't deleted.

Reply:
Okay, I will add that.

Reply:
Done in a65a31d.